test/protocol-tests/src/test/java/software/amazon/awssdk/protocol/tests/timeout/BaseApiCallAttemptTimeoutTest.java

```diff
@@ -15,126 +15,75 @@
 
 package software.amazon.awssdk.protocol.tests.timeout;
 
-import static com.github.tomakehurst.wiremock.client.WireMock.aResponse;
-import static com.github.tomakehurst.wiremock.client.WireMock.anyUrl;
-import static com.github.tomakehurst.wiremock.client.WireMock.post;
-import static com.github.tomakehurst.wiremock.client.WireMock.stubFor;
 import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
-import static software.amazon.awssdk.protocol.wiremock.WireMockUtils.verifyRequestCount;
 
-import com.github.tomakehurst.wiremock.stubbing.Scenario;
+import java.time.Duration;
 import org.junit.Test;
```

Comment:
Is it possible to switch these tests to Junit5 without too much work?

Reply:
Yeah, should be trivial. Will update